pkg/manifestclient/roundtripper.go

```diff
@@ -123,6 +125,24 @@ func (mrt *manifestRoundTripper) RoundTrip(req *http.Request) (*http.Response, e
 		// TODO handle label and field selectors
 		returnBody, returnErr = mrt.list(requestInfo)
 
+	case "watch":
+		// our watches do nothing.  We keep the connection alive (I  think), but nothing else.
```

Comment:
Listing all file based resources at start up can be interpreted either like creating or updating all of these for the first time. The code should generate corresponding create, resp. update events here for all the listable items. So the registered even handlers can react. Does the current roundtrip watch implementation lead to generating the events?

Reply:
That's an interesting idea.  We could probably better handle the "playback everything case" if we needed to.

Reply:
> Would those events not come from the list part of the listwatch concept within the informer?

A watch that requests initial events is not properly handled by this client. It may someday be an issue.